HIP/hip-0000-adding-pause–feature-to-HTS.md

```diff
@@ -0,0 +1,82 @@
+- hip: [TBD]
+- title: Pause feature on Hedera Token Service
+- author: chung@xfers.com, junhao@xfers.com, zhikai@xfers.com
+- type: Standards Track
+- category: Application
+- status: Draft
+- created: 2021-08-09
+- discussions-to: 
```

Comment:
Please add a discussion link to https://github.com/hashgraph/hedera-improvement-proposal/discussions/126